modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/ActiveActorTest.java

```diff
@@ -431,6 +435,7 @@ private TopologyAwareRaftGroupService startCluster(
     ) {
         when(msm.recoveryFinishedFuture()).thenReturn(completedFuture(0L));
         when(msm.invoke(any(), any(Operation.class), any(Operation.class))).thenReturn(completedFuture(true));
+        when(msm.getLocally(any(), anyLong())).then(invocation -> emptyMetastoreEntry());
```

Comment:
Why do you need to mock this?

Reply:
To avoid getting `null` in `Entry entry = msManager.getLocally(PLACEMENTDRIVER_LEASES_KEY, recoveryRevision);`, which will lead to NPE.